### ISSUE TITLE
Simplify the contribution guidelines 

### ISSUE THREAD

**Issue by tengqm** [kubernetes/website#28573]
The current contribution guideline is too wordy to contributors, old and new. As a reviewer myself, I have a difficult time to find out a guideline when attempting to quote it or refer a contributor to it. For example,

- We are not accepting "dual-hosted" or "3rd party" contents, where are those guide?
- We don't fix blog pages that are over 12 months old, where exactly is this mentioned?

If I just started reading the contribution guides, there are so many pages. The left hand navigator has no logic in it. The fonts are almost the same size. We have for "writing new topic" and "contributing new content", we have "reviewing changes" and "participating in SIG Docs", we have "Custom Hugo Shortcodes" and "advanced contributions", "content guide" listed alongside "content organization" and "page content types" ...

Let's consider how to tailor and reorganize this. Contributing to k8s docs should not be so difficult as it smells today.

**Comment by sftim** [comment]
Thanks @tengqm

/triage accepted
/sig contributor-experience
/priority important-longterm

**Comment by haripriya9647** [comment]
@tengqm @sftim I think we can add this point in the documentation that "we don't fix blog pages that are over 12 months old" and coming to the next point it's already mentioned as per the latest docs update. If you give me approval I will be happy to add a commit .

**Comment by vaibhav2107** [comment]
@haripriya9647 Are you still working on this issue?

**Comment by haripriya9647** [comment]
yes
